pkg/controller/jaeger/account.go

```diff
@@ -31,7 +31,8 @@ func (r *ReconcileJaeger) applyAccounts(ctx context.Context, jaeger v1.Jaeger, d
 	}
 
 	inv := inventory.ForAccounts(list.Items, desired)
-	for _, d := range inv.Create {
+	for i := range inv.Create {
+		d := inv.Create[i]
```

Comment:
why is that needed?

Reply:
Is gosec marking that here? I understand for others because it takes the address of the loop alias. But I don't get it why is marking this here